commands/htmlCapture.js

```diff
@@ -0,0 +1,48 @@
+
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+
+var mkdirpSync = function(path) {
+    try {
+        mkdirp.sync(path);
+    } catch(e) {
+        if ( e.code != 'EEXIST' ) throw e;
+    }
+}
+
+exports.command = function(fileName, callback) {
+    if (fileName && typeof fileName !== 'string') {
+        throw new Error('htmlCapture expects first parameter to be string; ' + typeof (fileName) + ' given');
+    }
+
+    var htmlCheck = /^\w+\.html$/;
```

Comment:
I was thinking about this, and I feel like having the ".html" be necessary makes the code more readable. ie: 
```javascript
browser
    .htmlCapture("home")
    .end();
```
doesn't imply functionality as well as
```javascript
    .htmlCapture("home.html")
```
Your thoughts?